formats/json-jackson/src/main/java/io/cloudevents/jackson/JsonCloudEventData.java

```diff
@@ -0,0 +1,47 @@
+package io.cloudevents.jackson;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.cloudevents.CloudEventData;
+
+import java.util.Objects;
+
+/**
+ * This class is a wrapper for Jackson {@link JsonNode} implementing the {@link CloudEventData}
+ */
+public class JsonCloudEventData implements CloudEventData {
+
+    private final JsonNode node;
+
+    public JsonCloudEventData(JsonNode node) {
+        this.node = node;
+    }
+
+    @Override
+    public byte[] toBytes() {
+        return node.toString().getBytes();
```

Comment:
Can you check if the node is null?

Reply:
It cannot be: if the `CloudEventData` is there, then there is something inside it. I guess I should add the check to the constructor of `JsonCloudEventData`

Reply:
Yes, that is what I meant.